### PR TITLE
Initial Upload

### PR DESCRIPTION
**Pull request by BenCoffeed**
Ok. Shiny new repo here.

### CODE REVIEW

File: .github/build-release.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        uses: actions/create-release@v1
+        id: create_release
+        with:
+          draft: false
+          prerelease: false
+          release_name: ${{ steps.version.outputs.version }}
+          tag_name: ${{ github.ref }}
+          body_path: CHANGELOG.md
+          artifacts:
+            - ./.tmp/rwell_bitdefender_amd.pkg
```
Comment (shadowhand) [inline .github/build-release.yml:41]: `amd` or `arm` ?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.tmp/
```
Comment (shadowhand) [inline .gitignore:1]: Why a dot-directory instead of something like `build/` ?

**Review by shadowhand — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (BenCoffeed) [reply .gitignore:1]: You assume there was a reason other than me just writing a script with `/tmp` in it and then saying "Ah. ./.tmp would be better." This is probably just from my history writing this sort of thing. Typically, I would dump this sort of thing in `/tmp` but figured I'd keep it all in one directory for CI sake. Then, I figured I'm ignoring it in GitHub. I may as well hide it in finder. Then `.tmp` was born.

Comment (shadowhand) [reply .gitignore:1]: Okay :)

**Comment by BenCoffeed** [comment]
@shadowhand should be good unless you think changing `.tmp` to `build/` or some such is significant. Happy to oblige if it still bothers you :)

File: Readme.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+## Requirements
+
+Since github actions automatically generates a github token that has enough permissions to draft a release, the only requirement is to set an environment secret of `BD_ID` which contains the short code denoting the org and deploy package-specific code within Gravityzone that determines the content of the install.xml file with the package download. You can find this by logging into your own BitDefender Gravityzone portal, going to `Network --> Packages`, checking the box next to the package you want to link to and then clicking on "Send download links" and looking at the macOS Downloader link. The link will look like `https://cloudgz.gravityzone.bitdefender.com/Packages/MAC/0/${BD_ID}/setup_downloader.dmg`. Copy the value where `${BD_ID}` is and store that as an environment secret. If you build a new package in Gravityzone, make sure to pick up the BD_ID of the new package and update the environment secret and re-deploy to get the package.
+
+## Outputs
+
+This process builds 4 artifacts. The 4 artifacts consist of 2 pairs of files:
+
+* `rwell_bitdefender_arm.pkg and rwell_bitdefender_arm.sig`
+* `rwell_bitdender_intel.pkg and rwell_bitdefender_intel.sig`
```
Comment (shadowhand) [inline Readme.md:14]:
```suggestion
* `rwell_bitdefender_arm.pkg` and `rwell_bitdefender_arm.sig`
* `rwell_bitdender_intel.pkg` and `rwell_bitdefender_intel.sig`
```

**Review by shadowhand — APPROVED** [review]
One more nit pick, but otherwise 💯

File: Readme.md (hunk 3 above)
Comment (BenCoffeed) [reply Readme.md:14]: Good call.

**Review by shadowhand — APPROVED** [review]

**Pull request merged**